README.md

```diff
@@ -1 +1,172 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://tailwindcss.com/docs/guides/create-react-app">Tailwind Css</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **List of NBA Teams with current conerence position**
+- **Statistics of each NBA team** 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://dev--marvelous-mandazi-c7fe36.netlify.app/)
+- [video walkthrough Link](https://www.loom.com/share/30fdc28e49524da69ffb00842d47eb26?sid=f617c166-ce16-4cd9-8177-1215b94615a4)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+- Clone this repo as described in the setup section. 
+- Make modifications as preferred
+
+### Prerequisites
+
+In order to run this project you need: To clone or fork and run it in a browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/migo1/nba-teams-stats.git
+```
+
+### Install
+
+- Run ` cd nba-teams-stats `
+- Run ` npm install `
+
+### Usage
+
+To run the project, execute the following command:
+
+- Run `npm start`
+
+### Run tests
+
+To run tests, run the following command:
+
+- to test validation errors `npx hint .`
+
+- to test CSS linter errors `npx stylelint "**/*.{css,scss}"`
+
+- to test JavaScript linter errors  `npx eslint .`
+
+- to run the jest tests `npx test`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **ROBERT MIGOT**
+
+- GitHub: [@migo1](https://github.com/migo1)
+- LinkedIn: [Robert Migot](https://www.linkedin.com/in/robert-m-3b3b1116a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add charts to display the statistics of each team
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/migo1/nba-teams-stats/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Thanks to all Microverse community
```

Comment:
As required in this milestone, I suggest you add an acknowledgment to the author of the design, [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio). In fulfillment of the requirement of the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/)😉😉.

<img width="1010" alt="image" src="https://github.com/migo1/nba-teams-stats/assets/64643995/0c881066-6289-43d7-aedc-752b6716a5ec">